desc/continuation.py

```diff
@@ -350,6 +350,8 @@ def _add_shaping(
             deltas["Rb_lmn"] *= bdry_step
         if "Zb_lmn" in deltas:
             deltas["Zb_lmn"] *= bdry_step
+        if "Rp_lmn" in deltas or "Zp_lmn" in deltas or "Lp_lmn" in deltas:
+            print("WARNING: Poincare BC is not yet supported for continuation")
```

Comment:
should raise an error here, not just print a warning (and if you actually want a warning use ``warnings.warn`` or ``desc.utils.warnif`` not just a print statement)

Reply:
Yes, it should be better. I wrote it this way to fall back to just perturbing the last closed flux surface boundary.